gnovm/stdlibs/std/emit_event.go

```diff
@@ -19,13 +19,26 @@ func X_emit(m *gno.Machine, typ string, attrs []string) {
 
 	_, pkgPath := currentRealm(m)
 	fnIdent := getPrevFunctionNameFromTarget(m, "Emit")
+	prevFunc := getPrevFunctionNameFromTarget(m, fnIdent) // get the previous function name of the function that called Emit
+
+	prevAddr, prevPath := X_getRealm(m, 1)
+	prev := prevStack{
+		PrevPkgPath: prevPath,
+		PrevPkgAddr: prevAddr,
+	}
+
+	ctx := m.Context.(ExecContext)
```

Comment:
Ctx is already initialized further down, make sure to drop the later init on L42

Reply:
252577668f81e62062e24f63f2feca98ea60dccc